langchain_tools_demo/orchestration/orchestration.py

```diff
@@ -0,0 +1,82 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from abc import ABC, abstractmethod
+from typing import Any, Dict, Generic, List, TypeVar
+
+import aiohttp
+
+# aiohttp context
+connector = None
+
+
+class classproperty:
+    def __init__(self, func):
+        self.fget = func
+
+    def __get__(self, instance, owner):
+        return self.fget(owner)
```

Comment:
I think we can use `@classmethod @property`: https://stackoverflow.com/questions/128573/using-property-on-classmethods/64738850#64738850

Reply:
Seems like this feature was removed in Python 3.13. :(